Radzen.Blazor/DataBoundFormComponent.cs

```diff
@@ -301,23 +301,24 @@ protected virtual IEnumerable View
         /// <returns>A Task representing the asynchronous operation.</returns>
         public override async Task SetParametersAsync(ParameterView parameters)
         {
+            // process all parameters first to prevent stale parameters
             var searchTextChanged = parameters.DidParameterChange(nameof(SearchText), SearchText);
+            var dataChanged = parameters.DidParameterChange(nameof(Data), Data);
+            var disabledChanged = parameters.DidParameterChange(nameof(Disabled), Disabled);
+            var result = base.SetParametersAsync(parameters);
+
             if (searchTextChanged)
             {
                 searchText = parameters.GetValueOrDefault<string>(SearchText);
```

Comment:
@enchev I am honestly quite confused by this line. I have not changed it, only moved it but I think it should say `nameof(SearchText)` instead of `SearchText` and was just ignored before as it was overriden by the `base.SetParametersAsync(..)` call? But I am also unsure why it even exists in the first place - `base.SetParametersAsync(..)` should already set the SearchText if it was supplied so there should be no reason to manually update the backing field.
Am I missing something here?